app/models/alchemy/content/factory.rb

```diff
@@ -122,12 +122,16 @@ def descriptions
       # Returns a normalized Essence type
       #
       # Adds Alchemy module name in front of given essence type
+      # unless there is a Class with the specified name that is an essence.
       #
       # @param [String]
       #   the essence type to normalize
       #
       def normalize_essence_type(essence_type)
         essence_type = essence_type.classify
+
+        return essence_type if is_an_essence?(essence_type)
```

Comment:
It turns out I don't even need line 15. If the user specifies the Alchemy namespaced version of the Essence, it will actually be an essence and the be returned as-is.